react-native/react/actions/login2.js

```diff
@@ -299,3 +306,123 @@ export function setDeviceName (name) {
     }, 1000)
   }
 }
+*/
+
+// Show a user/pass screen, call cb(user, passphrase) when done
+// title/subTitle to customize the screen
+export const askForUserPass = (title, subTitle, cb, hidePass = false) => {
+  return (dispatch) => {
+    dispatch({
+      type: Constants.actionAskUserPass,
+      title,
+      subTitle,
+      hidePass,
+      onSubmit: (username, passphrase) => {
+        dispatch({
+          type: Constants.actionSetUserPass,
+          username,
+          passphrase
+        })
+
+        cb()
+      }
+    })
+
+    dispatch(routeAppend({
```

Comment:
The mapStateToProps function will end up in the store, right? If so, this breaks @MarcoPolo's hot reloading.